### DISCUSSION TITLE
Revert back to the timeline editor?

### DISCUSSION THREAD

**Discussion by Usertyu** [olive-editor/olive#1865] [Q&A]
Is there a way to just revert back to the timeline editor in olive 0.2 or not? Just wondering

**Comment by Simran-B** [comment]
What do you mean by "revert back"? Is there a specific difference between 0.1 and 0.2 that you are unhappy with? Or is your question about something else?

**Reply by Usertyu** [reply]
I feel as tho the only way to add effects is to use the node base editor so I was wondering can I do those same effects just on the timeline based editor

**Reply by GUY-BBS** [reply]
To start, you can create dissolves in the timeline. You can make geometric wipes and moves by 'chequerboarding' alternate clips and adjusting position and size in the Parameter Editor using keyframes.

https://user-images.githubusercontent.com/49677111/155128606-98868d9e-76bd-425e-ae60-59169ca25963.mp4


https://user-images.githubusercontent.com/49677111/155128627-73c52499-8e51-4067-9acf-02330f2d99fd.mp4

**Reply by mamarilmanson** [reply]
I think it's pertaining to traditionally applying effects/transitions and adding adjustment layers similar to Premiere Pro/After Effects/Resolve... and maybe having the ability to do those mentioned along with the node editor that is in olive... (reference maybe how Resolve does the Editor Page and Fusion Page...) :)

**Comment by GUY-BBS** [comment]
Timeline layout as described above.
https://user-images.githubusercontent.com/49677111/155128935-53fe26a4-bea4-4ce0-bd67-de7337103cc4.mp4
